src/fields/StoreHoursField.php

```diff
@@ -57,6 +57,10 @@ public function getInputHtml($value, ElementInterface $element = null): string
      */
     public function serializeValue($value, ElementInterface $element = null)
     {
+        if (!$value) {
+            return null;
```

Comment:
Let’s move this to a separate PR. Each PR should be focussed on a single feature/enhancement/bugfix. Thanks!